go/cmd/dolt/commands/pull.go

```diff
@@ -18,6 +18,8 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/dolthub/dolt/go/libraries/doltcore/doltdb"
```

Comment:
(minor) you can move this down to line 25 or so to keep all the `dolthub/dolt/go/libraries` import lines together.